pkg/controller/worker/machines.go

```diff
@@ -268,3 +273,32 @@ func (w *workerDelegate) generateMachineConfig(ctx context.Context) error {
 
 	return nil
 }
+
+func (w *workerDelegate) enableAcceleratedNetworking(machineType, machineImageName, machineImageVersion string) bool {
+	// Check if used operating system and version are supporting accelerated networking.
+	var osSupport bool
+	for _, image := range w.cloudProfileConfig.MachineImages {
+		if image.Name != machineImageName {
```

Comment:
Hm, what if a machine image that is currently in use for shoots is removed from the `CloudProfile`? Then, in the next reconciliation the accelerated networking won't work anymore, right?

Reply:
Yep, exactly, but existing machine with accelerated networking wouldn't be affected.

Reply:
Hm, ok, wouldn't it? What does MCM when the respective field in the `AzureMachineClass` changes? Probably nothing... ok. Fine, resolving the conversation then.